java/clients/src/main/java/sleeper/clients/api/aws/AwsSleeperClientBuilder.java

```diff
@@ -65,7 +65,7 @@ public SleeperClient build() {
                 .tableIndex(tableIndex)
                 .tablePropertiesProvider(S3TableProperties.createProvider(instanceProperties, tableIndex, awsClients.s3()))
                 .tablePropertiesStore(S3TableProperties.createStore(instanceProperties, awsClients.s3(), awsClients.dynamo()))
-                .stateStoreProvider(StateStoreFactory.createProvider(instanceProperties, awsClients.s3(), awsClients.dynamo(), awsClients.s3TransferManager()))
+                .stateStoreProvider(StateStoreFactory.createProvider(instanceProperties, awsClients.s3(), awsClients.dynamo()))
```

Comment:
It looks like the field that was read here can be deleted (SleeperClientAwsClients.s3TransferManager).